applications/omnipresence/src/omnip_subscriptions.erl

```diff
@@ -298,6 +299,12 @@ handle_info({'timeout', Ref, ?EXPIRE_MESSAGE}=_R, #state{expire_ref=Ref, ready='
 handle_info(?TABLE_READY(_Tbl), State) ->
     lager:debug("recv table_ready for ~p", [_Tbl]),
     {'noreply', State#state{ready='true'}, 'hibernate'};
+handle_info('check_sync', #state{sync_nodes=[]} = State) ->
+    gen_listener:cast('omnipresence_shared_listener', {'ready'}),
```

Comment:
This should be a function call in omnipresence_shared_listener instead. The fact that it is a gen_listener should be hidden from external modules/processes.